src/test/e2e/helpers/database-init.ts

```diff
@@ -99,8 +99,7 @@ export default async function init(
 
     await db.raw(`DROP SCHEMA IF EXISTS ${config.db.schema} CASCADE`);
     await db.raw(`CREATE SCHEMA IF NOT EXISTS ${config.db.schema}`);
-    // @ts-expect-error
-    await migrateDb({ ...config, databaseSchema: config.db.schema });
```

Comment:
This `databaseSchema` property it's not needed: `migrateDb` destructures the object and takes only the `db` property: https://github.com/Unleash/unleash/blob/d4b40528d8926c996c671ffc080afdb88ca4ca25/src/migrator.ts#L8